pom.xml

```
@@ -228,7 +228,7 @@
         <jars.target.dir>${project.build.directory}/scala-${scala.binary.version}/jars</jars.target.dir>
 
         <maven.plugin.build.helper.version>3.3.0</maven.plugin.build.helper.version>
-        <maven.plugin.download.version>1.8.1</maven.plugin.download.version>
+        <maven.plugin.download.version>1.11.2</maven.plugin.download.version>
```

```suggestion
        <maven.plugin.download.version>1.11.3</maven.plugin.download.version>
```
1.11.3 has been just released, bring back the support for Java 8.
https://github.com/maven-download-plugin/maven-download-plugin/releases/tag/1.11.3